app/assets/stylesheets/application.css.scss.erb

```diff
@@ -27,6 +27,10 @@ body { padding-top: 60px; }
   margin-bottom: 100px;
 }
 
+table {
```

Comment:
No, I just want to style agent descriptions this time, so maybe we should make the code inside of .well.description this time.